.pre-commit-config.yaml

```diff
@@ -51,7 +51,7 @@ repos:
       # Run the formatter
     -   id: ruff-format
 -   repo: https://github.com/numpy/numpydoc
-    rev: v1.8.0
+    rev: v1.9.0
```

Comment:
Hi @skoudoro this was updated automatically and wasn't letting me go without updating it.

Let me know if it needs to be updated